src/emulator/gxm/src/gxm.cpp

```diff
@@ -145,13 +167,17 @@ static void output_glsl_parameters(std::ostream &glsl, const SceGxmProgram &prog
 static std::string generate_fragment_glsl(const SceGxmProgram &program) {
     GXM_PROFILE(__FUNCTION__);
 
+    LOG_DEBUG("Generating fragment shader");
+
     std::ostringstream glsl;
     glsl << "// Fragment shader.\n";
-    glsl << "#version 120\n";
+    glsl << "#version 140\n";
```

Comment:
macOS won't be able to load this yet.

Reply:
Ah yes. The rest of the code runs ok for me with 120, so let's try that on Mac too.